deployment/ansible/roles/model-my-watershed.rwd/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- stat: path={{ rwd_data_path }}
+  register: rwd_data
+- fail: msg="Data folder does not exist. Did you forget to mount /opt/rwd-data?"
```

Comment:
If we want the data available on the instance used to create an AMI, we'd have to add a `user_data` or `user_data_file` configuration to Packer. I'd lean toward avoiding that tight a coupling during AMI creation time, if we can.

See also: https://packer.io/docs/builders/amazon-ebs.html